app/models/sidebar_url.rb

```diff
@@ -53,6 +53,7 @@ class SidebarUrl < ActiveRecord::Base
       icon: "certificate",
       segment: SidebarUrl.segments["secondary"],
     },
+    { name: "Filter", path: "/filter", icon: "filter", segment: SidebarUrl.segments["secondary"] },
```

Comment:
I think you will need a migration to add this for existing sites + users, @yuriyaran did one recently for Messages

Reply:
agree, going to sort it out!